rhel/SPECS/clickhouse.spec

```diff
@@ -6,7 +6,7 @@
 
 Name:           percona-clickhouse
 Version:        19.7.5.27
-Release:        stable%{?dist}
+Release:        1.stable%{?dist}
```

Comment:
This may break `pmm-server` build, since [Clickhouse repository]() has no `19.7.5.27-1.stable` tag. It has only [19.7.5.27-stable](https://github.com/ClickHouse/ClickHouse/tree/v19.7.5.27-stable) tag.

Reply:
It looks like something is broken in [build/bin/build-server-rpm at PMM-2.0 branch](https://github.com/Percona-Lab/pmm-submodules/blob/PMM-2.0/build/bin/build-server-rpm), since it leads to the following error when `build/bin/build-server-rpm clickhouse` is invoked:

```
Cloning into 'ClickHouse-19.7.5.27-1.stable'...
+ cd ClickHouse-19.7.5.27-1.stable
+ echo '====> Checkout specific tag v19.7.5.27-1.stable'
+ git checkout v19.7.5.27-1.stable
====> Checkout specific tag v19.7.5.27-1.stable
error: pathspec 'v19.7.5.27-1.stable' did not match any file(s) known to git.
Makefile:8: recipe for target 'server' failed
make: *** [server] Error 1
```

It looks like the added `1.` prefix in the `Release` above isn't deleted properly from the tag during git checkout.

Reply:
Yes, you're right. I'll fix this ASAP.